### PR TITLE
Language switch

### PR DESCRIPTION
**Pull request by masself**
This PR enhances the Dynamo plugin with a language switch feature for books published in several languages. This feature is better than the native gitbook multilingual book: better integrated in UI, allows to switch language page-to-page, instead of just to the home page.

### CODE REVIEW

**Comment by Racel** [comment]
@masself - @QilongTang is the developer working on the primer. Please send him any PRs. 

@QilongTang - Please take a look.

File: README.md [hunk 1]
```diff
@@ -37,3 +38,79 @@ Default configuration is:
...
+- English: "switchLangLabel":"Languages"
+- German: "switchLangLabel":"Sprachen"
+- Japanese: "switchLangLabel":"言語"
+
+
+Example configuration for German:
+
+```js
+"dynamo": {
+      "languages":[
```
Comment (QilongTang) [inline README.md:88]: Where should this configuration reside?

**Comment by QilongTang** [comment]
One question on the language config, other code looks good to me

File: README.md (hunk 1 above)
Comment (masself) [inline README.md:88]: @QilongTang Thanks for looking. This config goes into the gitbook book.json. Book.json is where all plugin config params are defined. The "languages" object extends the dynamo plugin config, which had already "customLinks"  and "favicon". Thanks

Comment (QilongTang) [inline README.md:88]: @masself Thanks for the explanation. Does this mean once we merge this PR. There should be another PR in Primer to update the book.json there to utilize this new feature?

Comment (masself) [inline README.md:88]: The updated book.json is in branch Support_SP16. https://github.com/DynamoDS/DynamoPrimer/blob/Support_SP16/en/book.json
The branch Support_SP16 is a little confusing since in contains a lot of things (English content update, localization updates, book.json update, addition of language folders) but it does contain the necessary changes in en/book.json.

Comment (QilongTang) [inline README.md:88]: Great. I understand much better now. Good to merge.

Comment (masself) [inline README.md:88]: @QilongTang Will you merge? I don't have permission to merge in this repo.

Comment (QilongTang) [inline README.md:88]: @masself Merged.
